services/httpd/config_test.go

```diff
@@ -1,12 +1,65 @@
 package httpd_test
 
 import (
+	"reflect"
+	"strings"
 	"testing"
+	"text/template"
 
 	"github.com/BurntSushi/toml"
 	"github.com/influxdata/influxdb/services/httpd"
 )
 
+// TestConfig_ParseHeaders ensures we can properly extract the user supplied
+// headers from the configuration file.
+//
+// This test is a success if we get the data out as we put into the TOML
+// values.
+//
+func TestConfig_ParseHeaders(t *testing.T) {
```

Comment:
This covers config handling, but I'd also like to see a small [handler test](https://github.com/influxdata/influxdb/blob/master-1.x/services/httpd/handler_test.go) to show the headers are being added to a request (maybe something like this [test](https://github.com/influxdata/influxdb/blob/master-1.x/services/httpd/handler_test.go#L1729)).